bindings.c

```diff
@@ -1909,18 +1919,25 @@ int cg_open(const char *path, struct fuse_file_info *fi)
 
 int cg_access(const char *path, int mode)
 {
+	int ret;
 	const char *cgroup;
-	char *last = NULL, *path1, *path2, * cgdir = NULL, *controller;
+	char *path1, *path2, *controller;
+	char *last = NULL, *cgdir = NULL;
 	struct cgfs_files *k = NULL;
 	struct fuse_context *fc = fuse_get_context();
-	int ret;
+
+	if (strcmp(path, "/cgroup") == 0) {
+		if ((mode & W_OK) == 0)
```

Comment:
Actually, I'm not sure if this is really needed. Even if the cgroup root mount is writeable operations like `cg_mkdir()` et al. will fail with `EPERM`. Basically *all* operations will fail on the cgroup root mount point because they try to pick a controller from this path and if they do not find one they return an error. For example, `cg_mkdir()` does:

```
controller = pick_controller_from_path(fc, path);
if (!controller)
        return errno == ENOENT ? -EPERM : -errno;
```


My idea was that users should still be able to descend into `/var/lib/lxcfs/cgroup` but not be able to e.g. do `touch` or `mkdir`. So what do you think about removing the check here?